### ISSUE TITLE
LLM07 - website link still in markdown format

### ISSUE THREAD

**Issue by rossja** [OWASP/www-project-top-10-for-large-language-model-applications#266]
need to edit  the entry for llm07 on the llmtop10.com site such that the following link is html-ized:

```
Finally, LLM Plugin extensions can bring their own vulnerabilities. These are described in [LLM07 - Insecure Plugin Design](InsecurePluginDesign.md), 
```

**Comment by mkfnch** [comment]
@rossja Is this still an issue in @SClinton's [latest WordPress build](https://genaiowasp.wpcomstaging.com/llmrisk/llm07-insecure-plugin-design/)?

**Comment by SClinton** [comment]
Yup, he is going to hand it over to Jason at OWASP directly. Jason is out Today, so he may have sent the credentials. Once he does I’ve already given Jason the name server A records and IPs which he used with the other domains.  Should be a super quick edit.

 

From: Ads Dawson ***@***.***>
Reply-To: OWASP/www-project-top-10-for-large-language-model-applications ***@***.***>
Date: Monday, May 20, 2024 at 2:24 PM
To: OWASP/www-project-top-10-for-large-language-model-applications ***@***.***>
Cc: sclinton ***@***.***>, Mention ***@***.***>
Subject: Re: [OWASP/www-project-top-10-for-large-language-model-applications] LLM07 - website link still in markdown format (Issue #266)

 

@SClinton i thought that @mkfnch had an action item to redirect llmtop10.com
to the new site https://genai.owasp.org/?

> > I saw this, just a reminder that you Jason and Ads have edit rights for
> this.
>
Internal slack thread here
https://owasp.slack.com/archives/C05CM6DUZ1D/p1716083197637669 .. this is
preferred and also makes this GH issue redundant + any others related

**Comment by SClinton** [comment]
I assume mike does.

 

From: Ads Dawson ***@***.***>
Reply-To: OWASP/www-project-top-10-for-large-language-model-applications ***@***.***>
Date: Monday, May 20, 2024 at 2:33 PM
To: OWASP/www-project-top-10-for-large-language-model-applications ***@***.***>
Cc: sclinton ***@***.***>, Mention ***@***.***>
Subject: Re: [OWASP/www-project-top-10-for-large-language-model-applications] LLM07 - website link still in markdown format (Issue #266)

 

I saw this; just a reminder that you, Jason @rossja, and Ads @GangGreenTemperTatum have edit rights for this.

It needs to be done at the DNS layer which is part of access I don't have @SClinton

**Comment by mkfnch** [comment]
@SClinton I definitely don't have access to the DNS records of domains owned by OWASP.
